src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -203,7 +203,9 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
         val stønadType = dokumenttypeTilStønadType(søknad.dokumenttype) ?: return BehandlesAvApplikasjon.INFOTRYGD
         return if (finnesBehandlingINyLøsning(søknad.fnr, stønadType)) {
             BehandlesAvApplikasjon.EF_SAK
-        } else if (søknad.behandleINySaksbehandling && sakService.finnesIkkeIInfotrygd(søknad)) {
+        } else if (søknad.behandleINySaksbehandling &&
```

Comment:
Betyr det at vi også setter alle skolepengesøknader til ef, eller infotrygd. Har det noe å si?

Reply:
Dette skal ikke medføre noen endringer for skolepenger i prod. På søknader setter vi foreløpig `behandleINySaksbehandling = false` for skolepenger.